lib/phoenix_test/playwright.ex

```diff
@@ -358,6 +361,45 @@ defmodule PhoenixTest.Playwright do
     session
   end
 
+  @doc """
+  Add cookies to the browser context, using `Plug.Conn.put_resp_cookie/3`
+
+  Note that for signed cookies the signing salt is **not** configurable.
+  As such, this function is not appropriate for signed `Plug.Session` cookies.
+  For signed session cookies, use `add_session_cookie/3`
+
+  See `PhoenixTest.Playwright.CookieArgs` for the type of the cookie.
+  """
+  def add_cookies(session, cookies) do
+    tap(session, &BrowserContext.add_cookies(&1.context_id, cookies))
+  end
+
+  @doc """
+  Removes all cookies from the context
+  """
+  def clear_cookies(session, opts \\ []) do
+    tap(session, &BrowserContext.clear_cookies(&1.context_id, opts))
+  end
+
+  @doc """
+  Add a `Plug.Session` cookie to the browser context.
+
+  This is useful for emulating a logged-in user.
+
+  Note that that the cookie `:value` must be a map, since we are using
+  `Plug.Conn.put_session/3` to write each of value's key-value pairs
+  to the cookie.
+
+  The `session_options` are exactly the same as the opts used when
+  writing `plug Plug.Session` in your router/endpoint module.
+  """
+  def add_session_cookie(session, cookie, session_options) do
+    tap(session, fn session ->
+      cookie = CookieArgs.from_session_options(cookie, session_options)
+      post(guid: session.context_id, method: :add_cookies, params: %{cookies: [cookie]})
```

Comment:
Call `BrowserContext.add_cookies`

Reply:
I pushed this and it breaks a test. The problem is that `BrowserContext.add_cookies` does the `Plug.Conn.put_resp_cookie` signing, which uses a different signing key to `Plug.Session`.

I will add a new undocumented function to browser context that does just the post.

Reply:
Called the function `post_cookies`

Reply:
Thanks 👍

Reply:
I removed `CookieArgs` completely from `BrowserContext` and moved it into `Playwright.add_cookies`.

That way `BrowserContext` stays Playwright-specific and doesn't need to know about `Phoenix` cookie shape.
As a result, no need for separate `post_cookies` function.

Reply:
That feels better, for sure.